DOCS/man/options.rst

```diff
@@ -1905,9 +1905,12 @@ Audio
 
     .. admonition:: Warning
 
-        There is not much reason to use this. HDMI supports uncompressed
-        multichannel PCM, and mpv supports lossless DTS-HD decoding via
-        FFmpeg's new DCA decoder (based on libdcadec).
+        In general it is not recommended to use passthrough, since HDMI
+        supports uncompressed multichannel PCM, and mpv supports lossless
+        DTS-HD decoding via FFmpeg's new DCA decoder (based on libdcadec).
+        
+        Nevertheless, there are some cases in which it must be activated,
+        such as using 3D audio formats like Atmos or Auro-3D.
```

Comment:
```suggestion
        Nevertheless there are some cases in which it must be activated for correct playback,
        such as 3D audio formats like Dolby Atmos or Auro-3D.
```

Reply:
also a question: does ffmpeg even support auro-3d?

Reply:
Unfortunately Auro-3D was not the best example, since Auro-3D is a channel based codec like the other 2D audio formats. So it could potentially be decoded, though I'm not sure if ffmpeg can handle it. At least you would need more than 8 channels, which IMO would be the biggest restriction.

Dolby Atmos and DTS:X, on the other hand, are object based codecs. Which means in order to decode those signals, a decoder would need to know the current loudspeaker setup for correct channel assignment. Also more than 8 channels would be necessary.
So current best solution would be to let an AVR do the decoding.